lib/Monads/NonDetMonad_Sat.thy

```diff
@@ -28,6 +28,29 @@ definition ex_exs_validE ::
   "\<lbrace>P\<rbrace> f \<exists>\<lbrace>Q\<rbrace>, \<lbrace>E\<rbrace> \<equiv> \<lbrace>P\<rbrace> f  \<exists>\<lbrace>\<lambda>rv. case rv of Inl e \<Rightarrow> E e | Inr v \<Rightarrow> Q v\<rbrace>"
 
 
```

Comment:
nitpick: why two lines here, but one line before `subsection` below? suggest other way around

Reply:
oh, that's the local style. sad face.

Reply:
You're right, we should change that style, it's a bit strange.

Reply:
Style updated.